app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  class SessionsController < BaseAdminController
+    include AdminSessionTimeout
```

Comment:
This is already included in `BaseAdminController` so doesn't need to be included again.

Reply:
This is still here.

Reply:
Ah, I removed it from the public `SessionsController` (which is also legit), but not here